hack/test-local-registry.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+os::build::setup_env
+
+os::util::ensure::built_binary_exists 'dockerregistry'
+
+url="${REGISTRY_OPENSHIFT_SERVER_ADDR:-localhost:5000}"
+# find the first builder service account token
+token="$(oc get $(oc get secrets -o name | grep builder-token | head -n 1) --template '{{ .data.token }}' | os::util::base64decode)"
+echo
+echo "Login with:"
+echo "  docker login -p \"${token}\" -u user ${url}"
+echo
+
+REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY="${REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY:-/tmp/registry}" \
+  REGISTRY_OPENSHIFT_SERVER_ADDR="${url}" \
+	dockerregistry images/dockerregistry/config.yml
```

Comment:
i assume this expects a cluster to be running somewhere, how are the coordinates of the cluster being supplied to the registry?

Reply:
KUBECONFIG

Reply:
add a usage comment at the top of the file?

Reply:
Done